.github/workflows/flutter.yml

```diff
@@ -67,6 +67,7 @@ jobs:
           run: |
             cd flutter
             flutter pub get
+            flutter test --platform chrome
```

Comment:
lets also add the browser test to https://github.com/getsentry/sentry-dart/blob/main/flutter/tool/presubmit.sh#L12